lib/cucumber-api/steps.rb

```diff
@@ -27,10 +27,39 @@
   end
 end
 
+Then (/^I grab "([^"]+)"$/) do |k|
```

Comment:
I prefer to use `When I do something` for actions and `Then I assert something` for assertions. Is it okay with you?